spec/models/assignment_spec.rb

```diff
@@ -1706,26 +1706,6 @@ def grouping_count(groupings)
           expect(data.map { |h| h[:grace_credits_used] }.compact).to contain_exactly(1)
         end
       end
-
-      context 'there is an extra mark' do
```

Comment:
I reviewed the test changes more carefully. I'm not sure why you removed any of the tests in this file; it doesn't seem like they call the methods that you deleted.

Reply:
Hi David, thanks for checking - you're right. I must've focused on the keywords only.